### DISCUSSION TITLE
Problem with fixed and transform rotate

### DISCUSSION THREAD

**Discussion by Doliwa79** [tailwindlabs/tailwindcss#9244] [Help]
I have fixed navbar and rotated element on the page. When scrolling, rotated element overlap the navbar. This problem does not occur in bootstrap, only in tailwind.

**Comment by wongjn** [comment]
As per the [`transform` documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/transform):

> If the property has a value different than none, a [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) will be created.

To work around this, ensure the fixed navbar has a higher `z-index` than the rotated element.
